actions/publish-npm/main.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+cd llama-stack-client-typescript
+if [[ "$VERSION" == *"rc"* ]]; then
+  yarn version --new-version $VERSION -m "Bump version to $VERSION" --no-git-tag-version
+else
+  yarn version --new-version $VERSION -m "Bump version to $VERSION"
+fi
+yarn install
+yarn build
+cd dist
+if [[ "$VERSION" == *"rc"* ]]; then
+  yarn publish --tag rc --non-interactive --access public
+else
+  yarn publish --non-interactive --access public
+  git config --global user.email "github-actions@github.com"
+  git config --global user.name "GitHub Actions"
+  git commit -a -m "Bump version to $VERSION" --amend
+  git push origin HEAD:main --force
```

Comment:
oh, --force seems bad here? if main fails, someone did something concurrently and we should know

Reply:
make sense, update it to just git push. I wasn't able to test this part, will just wait for our next release to see if there's any issues.